frontend/src/queries/nodes/WebVitals/definitions.ts

```diff
@@ -66,10 +66,17 @@ export const getMetric = (
     metric: WebVitalsMetric,
     percentile: WebVitalsPercentile
 ): number | undefined => {
-    return results
-        ?.filter((result) => result.action.custom_name === metric)
-        .find((result) => result.action.math === percentile)
-        ?.data.slice(-1)[0]
+    const data = results
+        ?.filter((result) => result.action.custom_name === metric) // Filter to the right metric
+        .find((result) => result.action.math === percentile)?.data // Get the right percentile // Get the actual data array
+
+    // If CLS, just return last value, it can be 0
+    if (metric === 'CLS') {
+        return data?.slice(-1)[0]
```

Comment:
So if I have a non-zero CLS, it'll show 0 if I look on a day with no data?

Can we use NULL for no data, and then not need to special-case CLS?

Reply:
I'm using `/query` and I'm pretty positive I don't wanna mess with their calculations here, it would affect customers in ways I can't really predict. I'd prefer not to change it.

But to your point, yes, it'll display 0 even though it's actually "no data", but for CLS only.